plugins/module_utils/_vmware.py

```diff
@@ -32,54 +32,14 @@
     PYVMOMI_IMP_ERR = traceback.format_exc()
     HAS_PYVMOMI = False
 
-from ansible.module_utils.basic import env_fallback, missing_required_lib
+from ansible.module_utils.basic import missing_required_lib
 
 
 class ApiAccessError(Exception):
     def __init__(self, *args, **kwargs):
         super(ApiAccessError, self).__init__(*args, **kwargs)
 
 
-def vmware_argument_spec():
```

Comment:
@mikemorency Why didn't you also remove `vmware_client_argument_spec()` from `plugins/module_utils/_vmware_rest_client.py`?

Reply:
Good catch/suggestion!

I tried to not touch the rest client initially, and just stick to pyvmomi. Once i split everything up it made less sense to leave the rest client stuff alone for the docs/arg specs. I missed this, ill remove it